bciers/apps/reporting/src/app/layout.tsx

```diff
@@ -1,11 +1,33 @@
-import './global.css';
+/*
+A layout is UI that is shared between routes.
+The app directory must include a root app/layout.js.
+The root layout must define <html> and <body> tags.
+You should not manually add <head> tags such as <title> and <meta> to root layouts. Instead, you should use the Metadata API which automatically handles advanced requirements such as streaming and de-duplicating <head> elements.
+*/
+
+import "./global.css";
 import {
   metadata as regMetadata,
-  default as RegRootLayout,
   viewport as regViewport,
-} from 'registration/layout';
+} from "registration/layout";
+import { ThemeRegistry } from "@bciers/components";
 
 export const metadata = regMetadata;
 export const viewport = regViewport;
 
-export default RegRootLayout;
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en">
+      <body id="__next">
+        {
+          //👇️ provide MUI custom theme to the components within the layout
+        }
```

Comment:
Should we keep the Header and footer components here? It's missing a few things from the registration app layout

Reply:
I brought in just the one, as bringing in the header and footer becomes a chain of more and more components that need to be integrated. I'll create a tech-debt card to bring over the main layout components.